plugins/aws-proxy-filebased-credentials-provider/src/main/java/io/trino/aws/proxy/plugins/filebasedcredentialsprovider/FileBasedCredentials.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.aws.proxy.plugins.filebasedcredentialsprovider;
+
+import jakarta.validation.constraints.NotNull;
+
+public record FileBasedCredentials(@NotNull String emulatedAccessKey, @NotNull String emulatedSecret, String remoteAccessKey, String remoteSecret)
```

Comment:
We already have `Credentials`. This model isn't needed.

Reply:
The model is used for reading from a file where the credentials are listed in a different format. Credentials has just two fields - accessKey and secretKey. But, this one elaorates remote and emulated credentials

Reply:
Yes, I understand that. But, why is the file in a different format? Why not have the file use our existing `Credentials` model?